translator/config/schema.go

```diff
@@ -802,6 +808,25 @@ var schema = `{
             1827,
             3653
           ]
+        },
+        "filterDefinition": {
+          "type": "object",
+          "descriptions": "Define filters to apply to the log messages in this log file to determine whether to publish the message or not",
```

Comment:
Can circle back with a PM about these

Reply:
Got a green light on this. Also, I'm not sure this constitutes "customer-facing" in the same vein as the AWS public doc, since the schema is just used internally in the agent, as far as I know.

Reply:
schema.json is deployed in the RPM for customers to reference (I guess?).  So it's public-ish.